packages/vite-plugin/package.json

```
@@ -24,6 +24,6 @@
     "vite": "^2.7.0"
   },
   "peerDependencies": {
-    "vite": "^2.2.3 || ^3.0.0"
+    "vite": "^2.2.3 || ^3.0.0 || ^4.0.0"
```

From a compatibility perspective we suggest making `4.0.3` the peer dependency as there were [issues](https://github.com/vitejs/vite/pull/11409) with the initial release.

```suggestion
    "vite": "^2.2.3 || ^3.0.0 || ^4.0.3"
```